channels/applicationinsights-channel-js/Tests/Unit/src/Sender.tests.ts

```diff
@@ -1,20 +1,28 @@
 import { AITestClass } from "@microsoft/ai-test-framework";
 import { Sender } from "../../../src/Sender";
-import { Offline } from '../../../src/Offline';
+import { createOfflineListener, IOfflineListener } from '../../../src/Offline';
 import { EnvelopeCreator } from '../../../src/EnvelopeCreator';
 import { Exception, CtxTagKeys, Util } from "@microsoft/applicationinsights-common";
 import { ITelemetryItem, AppInsightsCore, ITelemetryPlugin, DiagnosticLogger, NotificationManager, SendRequestReason, _InternalMessageId, LoggingSeverity, getGlobalInst, getGlobal } from "@microsoft/applicationinsights-core-js";
 
 export class SenderTests extends AITestClass {
     private _sender: Sender;
     private _instrumentationKey = 'iKey';
+    private _offline: IOfflineListener;
 
     public testInitialize() {
         this._sender = new Sender();
         this._sender.initialize({ instrumentationKey: this._instrumentationKey }, new AppInsightsCore(), []);
+        this._offline = createOfflineListener("SenderTests");
     }
 
     public testCleanup() {
+        if (this._offline) {
+            this._offline.unload();
+        }
+        if (this._sender) {
+            //this._sender.unload();
```

Comment:
Maybe add a comment of what is preventing this one to be enabled if commented out on purpose

Reply:
This was an old change -- uncommenting and testing calling teardown (the new (old) name for unload as my initial code had both teardown and unload. While now there is only teardown.